### ISSUE TITLE
"Expecting double quoted ..."

### ISSUE THREAD

**Issue by nise** [HSU-ANT/beaqlejs#10]
All the config files are not correct. My Browser complains about commas after the objects in the "Testset" Array and after the key-value-pairs under "Files".

**Comment by timobaumann** [comment]
what is your browser? what precisely is the complaint?

**Comment by seebk** [comment]
Yes, please give more details if you intersted in a solution. If the [demo page](http://hsu-ant.github.io/beaqlejs/demo/mushra/) works in your browser there is probably a syntax error in your configuration file.

**Comment by nise** [comment]
Have look at the abx, mushra, and preference config:
`   {
      "Name": "Castanets",
      "TestID": "id2",
      "Files": {
        "A": "audio/castanets_ref.wav",
        "B": "audio/castanets_2.wav",
      }
    },
  ]
}`
The last coma leads to errors in Firefox.

**Comment by nise** [comment]
I would further suggest to define the config files as JSON, instead of putting a singe JS variable in such a file. 

In the index.html the JS code can be easily changed.
` window.onload = function () {
            $.ajax({
                url: '/config/ab-experiment-config.json', // change your config file here
                type: 'GET',
                dataType: 'json',
                success: function (data) {
                    console.log(data)
                    var TestConfig = data;
                    //testHandle = new MushraTest(TestConfig);    // <-- MUSHRA test class
                    testHandle = new AbxTest(TestConfig);     // <-- ABX test class

                    $('#Footer').prepend(testHandle.browserFeatureString() + '<br/>'); // still a bad coding practice here
                },
                error: function (err) {
                    console.error(err); //or whatever
                }
            });
        };`

**Comment by seebk** [comment]
> The last coma leads to errors in Firefox.

I agree that this comma is not required, but I do neither get any warnings nor errors in the Firefox web console window. I have just verified this with Firefox 60.5 ESR from my Debian system.
